jsonapi_requests/orm/api.py

```diff
@@ -0,0 +1,10 @@
+from jsonapi_requests.orm import registry
+
+
+class OrmApi:
+    def __init__(self, api):
```

Comment:
Maybe we will in the future make orm endpoint which speaks with orm objects instead of raw data. I would stick with this method.